futures-io/src/lib.rs

```diff
@@ -42,9 +45,11 @@ mod if_std {
 
     /// A type used to conditionally initialize buffers passed to `AsyncRead`
     /// methods, modeled after `std`.
+    #[cfg(feature = "read_initializer")]
     #[derive(Debug)]
     pub struct Initializer(bool);
```

Comment:
Is it better to re-export the std ones instead of our own ones? (The stability of this feature depends on it.)

Reply:
Done in https://github.com/rust-lang-nursery/futures-rs/pull/1845/commits/57686fd4726070f756a90dfba2c0c007846598ff